src/Factories/PageFactory.php

```diff
@@ -2,13 +2,14 @@
 
 namespace Astrotomic\Stancy\Factories;
 
+use Astrotomic\Stancy\Contracts\PageFactory as PageFactoryContract;
```

Comment:
Great that you've aliased it! 👍

Reply:
Checked how you did it in laravel-translatable before doing it here. Wanted to follow your code convention 👍